index.js

```diff
@@ -0,0 +1,36 @@
+const core = require("@actions/core");
+const tc = require("@actions/tool-cache");
+const os = require("os");
+
+(async function () {
+  try {
+    const version = core.getInput("version", { required: true });
```

Comment:
What happens exactly if there's no version? The old version had an explicit error message. Might be worth a negative test here.

Reply:
Currently this would result in a failed download...